.travis-build.sh

```diff
@@ -71,13 +75,13 @@ if [[ "${GROUP}" == "downstream" || "${GROUP}" == "all" ]]; then
   ##  * daikon-typecheck: (takes 2 hours)
 
   # checker-framework-inference: 18 minutes
-  (cd .. && git clone --depth 1 https://github.com/typetools/checker-framework-inference.git)
+  (cd .. && git clone --depth 1 https://github.com/${SLUGOWNER}/checker-framework-inference.git)
   export AFU=`pwd`/../annotation-tools/annotation-file-utilities
   export PATH=$AFU/scripts:$PATH
   (cd ../checker-framework-inference && gradle dist && ant -f tests.xml run-tests)
 
   # plume-lib-typecheck: 30 minutes
-  (cd .. && git clone https://github.com/mernst/plume-lib.git)
+  (cd .. && git clone https://github.com/${SLUGOWNER}/plume-lib.git)
```

Comment:
The typetools version should be the canonical version. I don't see a reason to use the personal repo instead of the organization repo like we do for everything else.